src/app-pages/post/PostDetailPage.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+
+import { usePostDetail } from '@/features/post/model/usePostDetailQuery';
+import { PostHeader } from '@/entities/post/ui/post-header/PostHeader';
+import { ActionBar } from '@/shared/ui/action-bar/ActionBar';
+import { PostBodySection } from '@/widgets/post-detail/PostBodySection';
+
+type PostDetailPageProps = {
+  postId: string;
+};
+
+export default function PostDetailPage({ postId }: PostDetailPageProps) {
+  const numericPostId = Number(postId);
+
+  const { data, isLoading, isError } = usePostDetail(numericPostId);
+
+  // postId 유효성 검증
+  if (isNaN(numericPostId) || numericPostId <= 0) {
+    return (
+      <div className="flex h-full w-full items-center justify-center">
+        <span>잘못된 게시글 ID입니다.</span>
+      </div>
+    );
+  }
+
+  if (isLoading)
+    return (
+      <div className="flex h-full w-full items-center justify-center">
+        <span>불러오는 중...</span>
+      </div>
+    );
+
+  if (isError || !data)
+    return (
+      <div className="flex h-full w-full items-center justify-center">
+        <span>게시글을 불러오지 못했습니다.</span>
+      </div>
+    );
+
+  const post = data;
+
+  return (
+    <div className="relative flex h-full w-full flex-col">
+      {/* 스크롤 영역 */}
+      <div className="flex-1 overflow-y-auto [-ms-overflow-style:none] [scrollbar-width:none] [&::-webkit-scrollbar]:hidden">
```

Comment:
스크롤 영역 없애는거 전역 스타일링으로 넣어놨어요!!
현재 develop에서 [src/shared/styles/globals.css] 경로에 scrollbar-hide 속성 만들어놔서 이걸로 대체해도 될 것 같아용
```suggestion
      {/* 스크롤 영역 */}
      <div className="flex-1 overflow-y-auto scrollbar-hide">
```

Reply:
오 감사합니다 사용하겠습니다!!